sormas-api/src/main/java/de/symeda/sormas/api/environment/WaterUse.java

```diff
@@ -4,13 +4,23 @@
 
 public enum WaterUse {
 
-	DRINKING_HOUSEHOLD_NEEDS,
-	RECREATION,
-	INDUSTRY_COMMERCE,
-	AGRICULTURE,
-	THERMOELECTRICITY_ENERGY,
-	OTHER,
-	UNKNOWN;
+	DRINKING_HOUSEHOLD_NEEDS(null),
+	RECREATION(null),
+	INDUSTRY_COMMERCE(null),
+	AGRICULTURE(null),
+	THERMOELECTRICITY_ENERGY(null),
+	OTHER(null),
+	UNKNOWN(null);
+
+	private WaterUse parent;
```

Comment:
parent is useless since none of the elements have a parent. please delete it

Reply:
removed